skript.js

```diff
@@ -0,0 +1,55 @@
+
+GETphoto("https://randomuser.me/api/?results=50")
+.then(
+  response=>alert("sucsess"),
+  error=>alert(`Rejected: ${error}`)
+);
+
+window.addEventListener("scroll", function(){
+    let block = document.getElementById('photo-area');
+    let counter = 1;
+    let contentHeight = block.offsetHeight;
+    let yOffset = window.pageYOffset;
+    let window_height = window.innerHeight;
+    let y = yOffset + window_height;
```

Comment:
Please avoid one letter variables. It's hard to understand what exactly this variable contains